stock_available_to_promise_release_block/models/stock_move.py

```diff
@@ -36,9 +36,11 @@ def _blocked_on_backorder(self):
 
     def action_block_release(self):
         """Block the release."""
-        if self.need_release:
-            self.release_blocked = True
+        for move in self:
+            if move.need_release:
+                move.release_blocked = True
 
     def action_unblock_release(self):
         """Unblock the release."""
-        self.release_blocked = False
+        for move in self:
+            move.release_blocked = False
```

Comment:
No need to change this implementation, `records.field = value` already set `value` on `field` for all `records`.